visualization/dashboard/src/MLIDashboard/Cohort.ts

```diff
@@ -0,0 +1,128 @@
+import { IFilter, FilterMethods } from "./Interfaces/IFilter";
+import { JointDataset } from "./JointDataset";
+import { ModelExplanationUtils } from "./ModelExplanationUtils";
+
+export class Cohort {
+    public static CohortKey: string = "Cohort";
+    private static _cohortIndex: number = 0;
+
+    public rowCount: number = 0;
+    private readonly cohortIndex: number;
+    private mutateCount: number = 0;
+    private _filteredData: Array<{[key: string]: number}>;
+    private _cachedAverageImportance: number[];
+    private _cachedTransposedLocalFeatureImportances: number[][];
+    private currentSortKey: string | undefined;
+    private currentSortReversed: boolean = false;
+    constructor(public name: string, private jointDataset: JointDataset, public filters: IFilter[] = []) {
+        this.cohortIndex = Cohort._cohortIndex;
+        this.name = name;
+        Cohort._cohortIndex += 1;
```

Comment:
what does the cohort index represent?  some documentation would be useful here... not sure if python-style docs can be added to typescript

Reply:
yes, its a static property on cohorts, the idea is to have a way of tracking uniqueness, like an ID.